docs/reference/buildx_history_export.md

```diff
@@ -15,3 +15,57 @@ Export a build into Docker Desktop bundle
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Export one or more build records to `.dockerbuild` archive files. These archives
+contain metadata, logs, and build outputs, and can be imported into Docker
+Desktop or shared across environments.
+
+## Examples
+
+### <a name="export-single-build"></a> Export a single build to a custom file
+
+```console
+docker buildx history export qu2gsuo8ejqrwdfii23xkkckt --output mybuild.dockerbuild
+```
+
+You can find build IDs by running:
+
+```console
+docker buildx history ls
+```
+
+### <a name="export-multiple-builds"></a> Export multiple builds to individual `.dockerbuild` files
```

Comment:
Why show this if it is effectively just running the above command two times? I could see "Export multiple builds to single `.dockerbuild` file" being different but not this one.

Reply:
Yes I think it should be smth like:

```
docker buildx history export qu2gsuo8ejqrwdfii23xkkckt qsiifiuf1ad9pa9qvppc0z1l3 -o multi.dockerbuild

docker buildx history export ^1 ^2 -o multi.dockerbuild
```